src/main/java/uk/gov/hmcts/divorce/legaladvisor/event/LegalAdvisorMakeDecision.java

```diff
@@ -136,6 +137,8 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
         final CaseData caseData = details.getData();
         final ConditionalOrder conditionalOrder = caseData.getConditionalOrder();
 
+        caseData.getConditionalOrder().setIsAdminClarificationSubmitted(YesOrNo.NO);
```

Comment:
should this be set to null rather than No, since we're just resetting (I assume)